apps/webmin.sh

```diff
@@ -66,7 +66,7 @@ print_text_in_color "$ICyan" "Configuring Webmin..."
 # redirect access on http to https
 check_command systemctl stop webmin
 ## TODO: This will cause Webmin to redirect to the hostname 'nextcloud' and not the IP address if on LAN
-check_command sed -i "s|ssl_redirect=.*|ssl_redirect=1|g" /etc/webmin/miniserv.conf
```

Comment:
BTW: what does the g at the end of the sed command do?

Reply:
https://unix.stackexchange.com/questions/484659/what-does-this-command-do-sed-s-g

I've always used it and it works, never digged down in the details. :)

Reply:
I've just asking because I've seen this command also without the g...